docs/05-technical-reference/evnt-01-event-types.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Event types
+---
+
+Eventing supports the following event types:
+- [Cloud Events](https://cloudevents.io/) - they use a specification for describing event data in a common way.
+- legacy events - they are converted to Cloud Events by [Event Publisher Proxy](./00-architecture/evnt-01-architecture.md#event-publisher-proxy), which also adds a `sap.kyma.custom` prefix.
+
+## Event form
```

Comment:
(nit) In code, its referred to as `Event Type Format`, or maybe we can refer it as `Event Name Format`?